docs/source/write_first_app.rst

```diff
@@ -344,17 +366,18 @@ interacts with the ledger.
 
   async queryAllCars(ctx) {
 
-    const startKey = 'CAR0';
-    const endKey = 'CAR999';
```

Comment:
start/end key value got changed in the javascript/go/java/typescript sample. This  reflects the latest code changes in the queryAllCars()